.github/workflows/generate-config.yaml

```
@@ -0,0 +1,133 @@
+name: Update config
+
+on:
+  pull_request_target:
+    types: [closed]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-config:
+    runs-on: ubuntu-latest
+
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      CHAINS_PAT: ${{ secrets.CHAINS_PAT }}
+      EXTERNAL_REPO: ${{ secrets.EXTERNAL_REPO }}
+
+    steps:
+      - name: Checkout axelar-contract-deployments repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+
+      - name: Check if PR was merged
+        id: check_if_merged
+        run: |
+          if [ "${{ github.event.pull_request.merged }}" != "true" ]; then
+            echo "PR was not merged. Exiting."
+            exit 0
+          fi
+
+      - name: Get the merge commit SHA
+        id: merge_commit
+        run: |
+          MERGE_COMMIT=$(git log -1 --pretty=format:"%H")
+          echo "MERGE_COMMIT=$MERGE_COMMIT" >> $GITHUB_ENV
+
+      - name: Determine NETWORKS
+        id: determine_networks
+        run: |
+          echo "Checking modified files in commit ${{ env.MERGE_COMMIT }}"
+          MODIFIED_FILES=$(git diff --name-only HEAD~1..HEAD)
+          echo "Modified files: $MODIFIED_FILES"
+
+          NETWORKS=()
+
+          if echo "$MODIFIED_FILES" | grep -q "axelar-chains-config/info/mainnet.json"; then
+            NETWORKS+=("mainnet")
+          fi
+          if echo "$MODIFIED_FILES" | grep -q "axelar-chains-config/info/testnet.json"; then
+            NETWORKS+=("testnet")
+          fi
+          if echo "$MODIFIED_FILES" | grep -q "axelar-chains-config/info/stagenet.json"; then
+            NETWORKS+=("stagenet")
+          fi
+          if echo "$MODIFIED_FILES" | grep -q "axelar-chains-config/info/devnet-amplifier.json"; then
+            NETWORKS+=("devnet-amplifier")
+          fi
+
+          if [ ${#NETWORKS[@]} -eq 0 ]; then
+            echo "No relevant changes detected."
+            echo "NETWORKS=none" >> $GITHUB_ENV
+            exit 0
+          fi
+
+          echo "Detected networks: ${NETWORKS[*]}"
+          echo "NETWORKS=${NETWORKS[*]}" >> $GITHUB_ENV
+
+      - name: Checkout external repository
+        if: env.NETWORKS != 'none'
```

```suggestion
        if: env.NETWORKS != ''
```
does this work? the variable isn't set to `none`